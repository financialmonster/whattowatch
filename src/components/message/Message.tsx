import React, { FC } from 'react';
import styled from 'styled-components';

type TMessageProps = {
    children: string;
}

const MessageWrapper = styled.div`
    min-height: 330px;
    padding-top: 180px;
    padding-left: 50px;
    padding-right: 50px;
    text-align: center;
    font-size: 32px;
    width: 1140px;
    margin: 0 auto;
    line-height: 40px;
`;

export const Message: FC<TMessageProps> = ({children}) => {
    return (
        <MessageWrapper>
            { children }
        </MessageWrapper>
    );
}
