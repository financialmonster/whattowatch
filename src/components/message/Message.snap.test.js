import React from 'react';
import renderer from 'react-test-renderer';

import { Message } from './Message';

it(`ErrorMessage: correctly rendered`, () => {
    const tree = renderer.create(
        <Message>
            Test
        </Message>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
