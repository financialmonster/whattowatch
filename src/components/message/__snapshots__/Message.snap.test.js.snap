// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage: correctly rendered 1`] = `
<div
  className="sc-AxjAm cqcwNi"
>
  Test
</div>
`;
